development/docker-compose.yml

```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   roscore:
-    image: usdotfhwastoldev/carma-base:develop
+    image: usdotfhwastol/carma-base:carma-system-4.5.0
```

Comment:
ORG and tag here needs to change. all occurrences in this file.

Reply:
Done